.github/workflows/dapr.yml

```diff
@@ -44,20 +44,22 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        os: [ubuntu-latest, windows-2019, macOS-latest]
+        os: [ubuntu-latest, macOS-latest]
         target_arch: [arm, arm64, amd64]
         include:
           - os: ubuntu-latest
             target_os: linux
           - os: windows-2019
```

Comment:
I think for CI we can stick to one Windows runner only. Thoughts @artursouza ?

Reply:
We can remove windows-2019 when we deprecate 1809 image, or we can do it right away in this PR as well.

Reply:
Let's keep 2019 for now. We should announce this deprecation in 1.11 and stop publishing in 2 releases.